.travis.yml

```diff
@@ -19,3 +19,6 @@ install:
 
 script:
 - travis_retry npm test
+- travis_retry npm test
```

Comment:
why test twice?

Reply:
Whups, that's a typo - sorry, got pretty late yesterday 😬

Reply:
👍